app/build.gradle

```diff
@@ -66,6 +67,7 @@ dependencies {
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
Are you using constraint layout anywhere? if no then remove it

Reply:
Ok (Added by Studio when I created new activity for Notification)